src/e2e/java/uk/gov/hmcts/reform/sscs/functional/sya/notifications/WelshNotificationsFunctionalTest.java

```diff
@@ -177,12 +113,6 @@
     @Value("${notification.welsh.appealCreated.appointee.emailId}")
     private String appealCreatedAppointeeEmailIdWelsh;
 
-    @Value("${notification.welsh.hearingAdjourned.appointee.emailId}")
```

Comment:
Why we hae to remove the hearingAdjouned one here?

Reply:
It was an un-used variable.